{{cookiecutter.project_slug}}/clients/vue3/src/views/Home.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div class="flex flex-col items-center">
     <img alt="Project Logo" src="../assets/logo.png" />
-    <h1>Welcome to {{cookiecutter.project_name}}!</h1>
+    <h1 class="heading">Welcome to {{ cookiecutter.project_name }}!</h1>
```

Comment:
**Info:** Added the `heading` custom class we had already implemented in PR #94 to this `<h1>` heading tag, as we want to help visually emphasize that it's an h1 heading (since Tailwind inherently removes the heading tags default stylings)